bridge/opencensus/metric.go

```diff
@@ -38,26 +42,57 @@ type exporter struct {
 }
 
 // NewMetricExporter returns an OpenCensus exporter that exports to an
-// OpenTelemetry exporter.
+// OpenTelemetry (push) exporter.
 func NewMetricExporter(base metric.Exporter, res *resource.Resource) metricexport.Exporter {
-	return &exporter{base: base}
+	return &exporter{base: base, res: res}
 }
 
 // ExportMetrics implements the OpenCensus metric Exporter interface by sending
 // to an OpenTelemetry exporter.
 func (e *exporter) ExportMetrics(ctx context.Context, ocmetrics []*ocmetricdata.Metric) error {
 	otelmetrics, err := internal.ConvertMetrics(ocmetrics)
 	if err != nil {
-		return err
+		otel.Handle(err)
+	}
+	if len(otelmetrics) == 0 {
+		return nil
 	}
 	return e.base.Export(ctx, metricdata.ResourceMetrics{
 		Resource: e.res,
 		ScopeMetrics: []metricdata.ScopeMetrics{
 			{
 				Scope: instrumentation.Scope{
-					Name: "go.opentelemetry.io/otel/bridge/opencensus",
+					Name: scopeName,
 				},
 				Metrics: otelmetrics,
 			},
 		}})
 }
+
+type reader struct {
+	manager *metricproducer.Manager
+}
+
+// NewBridge returns a metric.Bridge that can be used as
+// a source of metrics for OpenTelemetry Pull exporters.
+func NewBridge() metric.Bridge {
+	return &reader{
+		manager: metricproducer.GlobalManager(),
+	}
+}
+
+// Collect overrides the manual reader's collect method to retrieve metrics from OpenCensus.
```

Comment:
Update this docs since this is implementing the`metric.Bridge` interface now.

Reply:
done